Misc/NEWS.d/next/Tests/2019-04-01-17-44-36.bpo-31904.32Ef3G.rst

```diff
@@ -0,0 +1 @@
+Skip test case in test_cmd_line for VxWorks doesn't support preexec when do Popen
```

Comment:
Honestly, I'm not sure that it's worth it to document invidivual tests changes. Can you please remove the NEWS entry? I will simply add the GitHub label "skip news".